src/utils/soknad-felles/SoknadStatustekst.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { tilLesbarDatoMedArstall } from "../datoUtils";
+import { formaterOrgnr } from "@/utils";
+import {
+  Soknadstatus,
+  SykepengesoknadDTO,
+} from "@/data/sykepengesoknad/types/SykepengesoknadDTO";
+import { BodyShort } from "@navikt/ds-react";
+
+const texts = {
+  korrigert: "Korrigert",
+};
+
+interface SoknadStatustekstProps {
+  soknad: SykepengesoknadDTO;
+}
+
+const SoknadStatustekst = ({ soknad }: SoknadStatustekstProps) => {
+  const isSoknadSentToNav = !!soknad.sendtTilNAVDato;
+  const isSoknadSentToArbeidsgiver = !!soknad.sendtTilArbeidsgiverDato;
+
+  const arbeidsgiver = soknad.arbeidsgiver?.navn;
+  const orgnr = soknad.arbeidsgiver?.orgnummer
+    ? formaterOrgnr(soknad.arbeidsgiver.orgnummer)
+    : null;
```

Comment:
Ikke veldig viktig å endre, bare en alternativ måte som kanskje er cleanere
```suggestion
  const orgnr = soknad.arbeidsgiver?.orgnummer && formaterOrgnr(soknad.arbeidsgiver.orgnummer);
```